.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Interesting that this file & the `PULL_REQUEST_TEMPLATE` ended up in this PR... I wonder if `rebase` was actually the move!

Reply:
That's a good point Jake, I wonder also if it had to do with me undoing one too many commits when making the branch.